admin/metabox/class-metabox.php

```diff
@@ -230,10 +230,26 @@ public function localize_replace_vars_script() {
 		return array(
 			'no_parent_text' => __( '(no parent)', 'wordpress-seo' ),
 			'replace_vars'   => $this->get_replace_vars(),
+			'scope'          => $this->determine_scope(),
 		);
 	}
 
 	/**
+	 * Determine whether the current URL is a page or a post.
```

Comment:
The eventual scope has no relation to the URL of the page so the description should be something like: Voorbeeld! `Determine the scope on which the JavaScript can base which replacement variables should be replaced.`